### DISCUSSION TITLE
How do I apply arguments properly ?

### DISCUSSION THREAD

**Discussion by olmohake** [postgresml/postgresml#1517] [Q&A]
Thanks for the amazing work! I am trying to build a prof of concept using huggingface models that might be replaced with custom build once later on. [eleldar/theme-classification](https://huggingface.co/eleldar/theme-classification?candidate_labels=relationship%2Cwork%2C+friendship%2C+hobby&multi_class=true&text=Lets+see+how+much+work+i+will+get+done+%5E%5E) supports the arguments multi_class and candidate_labels but I am currently running into problems applying them correctly.
```sql
topics JSONB GENERATED ALWAYS AS (pgml.transform(
    	task   => '{
	       "task": "text-classification", 
               "model": "eleldar/theme-classification"
        }'::JSONB,
        args => '{ 
              "multi_class": true, 
              "candidate_labels": "[a, b,c]"
        }'::JSONB,
    	inputs => ARRAY[content]
)) STORED
```
This results in:

> unexpected keyword argument 'multi_class'

What am i missing?

**Comment by montanalow** [comment]
To accept the multi_class + candidate_labels arguments, we'll need to add support for the zero shot classification pipeline. 

```
from transformers import pipeline
classifier = pipeline("zero-shot-classification",
                      model="facebook/bart-large-mnli")
```

https://huggingface.co/eleldar/theme-classification

**Reply by montanalow** [reply]
Issue #1519
